logproxy-client/src/main/java/com/oceanbase/clogproxy/client/connection/ClientHandlerV01.java

```diff
@@ -0,0 +1,429 @@
+/* Copyright (c) 2021 OceanBase and/or its affiliates. All rights reserved.
+oblogclient is licensed under Mulan PSL v2.
+You can use this software according to the terms and conditions of the Mulan PSL v2.
+You may obtain a copy of Mulan PSL v2 at:
+         http://license.coscl.org.cn/MulanPSL2
+THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+See the Mulan PSL v2 for more details. */
+
+package com.oceanbase.clogproxy.client.connection;
+
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.oceanbase.clogproxy.client.config.ClientConf;
+import com.oceanbase.clogproxy.client.enums.ErrorCode;
+import com.oceanbase.clogproxy.client.exception.LogProxyClientException;
+import com.oceanbase.clogproxy.common.packet.CompressType;
+import com.oceanbase.clogproxy.common.packet.HeaderType;
+import com.oceanbase.clogproxy.common.packet.ProtocolVersion;
+import com.oceanbase.clogproxy.common.packet.protocol.LogProxyProto;
+import com.oceanbase.clogproxy.common.packet.protocol.V1Proto;
+import com.oceanbase.clogproxy.common.util.NetworkUtil;
+import com.oceanbase.oms.logmessage.LogMessage;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.ByteBufAllocator;
+import io.netty.buffer.ByteBufUtil;
+import java.util.concurrent.BlockingQueue;
+import net.jpountz.lz4.LZ4FastDecompressor;
+import org.apache.commons.lang3.Conversion;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ClientHandlerV01 {
```

Comment:
There are `v1` in proto file but `V01` in class file, I think it's better to use the same suffix.
And for now I can't quickly find out which version is the newer one from their name, maybe we can use a more understandable suffix?